static/flow/views/data-set-view.js

```diff
@@ -81,7 +81,7 @@ var DataSetView = function(options) {
     detailsDiv.append(endTime);
 
 
-    var canvas = $('<canvas>', { id: 'data-set-canvas' } );
```

Comment:
Another good tech-debt story is unifying spacing and line breaks around all of this tangled jQuery.  I'm not a fan of some of the cases where we pack lengthy jQuery statements onto a single line (but stuck with the pattern since that was the precedent).